### PR TITLE
Convert README to reStructuredText

### LINKED ISSUE #521: Fix Readme rendering on PyPI
Currently, PyPI does not support Markdown. As a result, the [description for mtools](https://pypi.python.org/pypi/mtools) is not rendered correctly.

I suggest one of two fixes:

- convert `README.md` to reStructuredText, which renders correctly on both PyPI and GitHub
- specify the [Description-Content-Type](https://packaging.python.org/specifications/#description-content-type) as `text/markdown` in `setup.py` using [long_description_content_type](https://github.com/pypa/setuptools/pull/1075). Note this option has not yet been implemented in [readme_renderer](https://github.com/pypa/readme_renderer), so currently will not work.

I'm happy to do the conversion work if that is the preferred path.

### PR DESCRIPTION
**Pull request by kallimachos**
- convert README.md to README.rst, as reStructuredText renders correctly on both PyPI and GitHub
- add Python 2.7 badge

Closes #521

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+======
+
+|PyPI version| |Build Status| |Python 27|
+
+**mtools** is a collection of helper scripts to parse, filter, and visualize
+MongoDB log files (``mongod``, ``mongos``). mtools also includes ``mlaunch``, a
+utility to quickly set up complex MongoDB test environments on a local machine.
+
+.. figure:: ./mtools.png
+   :alt: mtools box
```
Comment (stennie) [inline README.rst]: GitHub is rendering the alt text after the image: https://github.com/kallimachos/mtools/blob/4fa811efd1f28a46d823e9df19fb62e548cbf32d/README.rst

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+What's in the box?
+------------------
+
+The following tools are in the mtools collection:
+
+`mlogfilter <https://github.com/rueckstiess/mtools/wiki/mlogfilter>`__ slices
```
Comment (stennie) [inline README.rst]: In MarkDown the tool description appears on a line below the tool name. With GitHub RST rendering the description follows immediately after the tool name, which doesn't flow well. Can you add delimiters or newlines?

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Disclaimer
+----------
+
+This software is not supported by `MongoDB, Inc. <https://www.mongodb.com>`__
+under any of their commercial support subscriptions or otherwise. Any usage of
+mtools is at your own risk. Bug reports, feature requests and questions can be
+posted in the `Issues
+<https://github.com/rueckstiess/mtools/issues?state=open>`__ section here on
+github.
```
Comment (stennie) [inline README.rst]: Tweak to "GitHub" while we're tidying up :)

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+for instructions.
+
+Disclaimer
+----------
+
+This software is not supported by `MongoDB, Inc. <https://www.mongodb.com>`__
+under any of their commercial support subscriptions or otherwise. Any usage of
+mtools is at your own risk. Bug reports, feature requests and questions can be
+posted in the `Issues
+<https://github.com/rueckstiess/mtools/issues?state=open>`__ section here on
```
Comment (stennie) [inline README.rst]: "here on" => "on" (since this readme will be displayed in other contexts like PyPi)

File: README.rst (hunk 2 above)
Comment (kallimachos) [reply README.rst]: Yep. A definition list should work nicely here.

File: README.rst (hunk 4 above)
Comment (kallimachos) [reply README.rst]: Fixed.

File: README.rst (hunk 3 above)
Comment (kallimachos) [reply README.rst]: Fixed.

File: README.rst (hunk 1 above)
Comment (kallimachos) [reply README.rst]: Hrm. Originally there was a `<p>` with that text in it that I removed in an additional commit. I think you might be looking at an outdated version. I'll push the other changes and then check.

Comment (kallimachos) [reply README.rst]: I don't see an extra text after the image in the latest render. Could you please confirm?

**Pull request merged**